test.sh

```diff
@@ -35,11 +33,21 @@ docker-compose build
 # unpack latest build of buildpack
 docker-compose run --rm tester bash ./unpack.sh
 
-# run tests against latest build of buildpack
-docker-compose run --rm \
- -w "$BUILDPACK_ROOT_DIR/ci" \
- tester cucumber \
- --format pretty \
- --format junit \
- --out ./features/reports \
- ${INTEGRATION_TAG}
+if [ -n "$CF_API_ENDPOINT" ]; then
+     # run tests against latest build of buildpack
```

Comment:
this is 100% an ignorable nit but now you've capitalized one comment and I want it for all the others too
```suggestion
     # Run tests against latest build of buildpack (including integration tests against remote foundation)
```

Reply:
@BradleyBoutcher I still think it'd be helpful to add something like I suggested to the end of this line
> (including integration tests against remote foundation)

to clarify that in this flow, we are also running those tests

Reply:
Ah thank you! I didn't see the rest of this change and only capitalized the letter. I'll commit this change 👍